src/modules/shared/components/Html/Html.tsx

```diff
@@ -15,7 +15,7 @@ export interface HtmlProps {
 const Html: FunctionComponent<HtmlProps> = ({
 	content,
 	sanitizePreset = RICH_TEXT_SANITIZATION,
-	type = 'p',
+	type = 'div',
```

Comment:
Make sure to check styles of components that implement `Html` for any selectors relying on the previous `p`-tag default.